subset/COYO-Labeled-300M/download/dataproc/dataproc-initialization.sh

```diff
@@ -3,7 +3,7 @@
 # Install img2dataset
 pip install --upgrade gcsfs
 pip install tensorflow tensorflow_io
-pip install git+https://github.com/justHungryMan/img2dataset
+pip install img2dataset
```

Comment:
```suggestion
pip install img2dataset>=1.34.0
pip install --upgrade gcsfs
pip install tensorflow tensorflow_io
```

gcsfs must be upgraded after installing img2dataset.